larray/io/session.py

```diff
@@ -173,6 +173,7 @@ def _dump(self, key, value, *args, **kwargs):
 
     def close(self):
         self.handle.close()
+        self._delete_temporary_file()
```

Comment:
Unsure doing this in close is a good thing. These are two separate operations. Wouldn't it be better to call the method from dump_arrays instead?